hw/ip/otbn/doc/otbn_blockarch.svg

```diff
@@ -8,13 +8,13 @@
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
    version="1.1"
-   viewBox="0 0 949.50442 767.90681"
+   viewBox="0 0 1076.6736 886.2356"
```

Comment:
(This comment applies to this file but not exactly to this line.)

I think the lines for the randomness returned by EDN and the sideloaded key _outside_ `otbn_core` should be colored the same as inside (green if I'm not mistaken)